tutorials/notebooks/astropy-coordinates/3-Coordinates-Velocities.ipynb

```diff
@@ -0,0 +1,529 @@
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "test_coord = SkyCoord(\n",
+    "    ra=10*u.deg, \n",
+    "    dec=20*u.deg,\n",
+    "    pm_ra_cosdec=1*u.mas/u.yr,\n",
+    "    pm_dec=2*u.mas/u.yr)\n",
+    "\n",
+    "# This cell will raise an exception - this is expected!\n",
+    "test_coord.transform_to(coord.Galactocentric())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evolving Coordinate Positions Between Epochs\n",
+    "\n",
+    "For nearby or fast-moving stars, a star's position could change appreciably between two well-spaced observations of the source. For such cases, it might be necessary to compute the position of the star at a given time using the proper motion or velocity of the star. Let's demonstrate this idea by comparing the sky position of a source as measured by [*Gaia* Data Release 2](https://www.cosmos.esa.int/web/gaia/dr2) (given at the epoch J2015.5) to an image near this source from the Digitized Sky Survey (DSS; digital scans of photographic plates observed in the 1950s). \n",
+    "\n",
+    "From previous astrometric measurements, we know that the star HD 219829 has very large proper motion: Close to 0.5 arcsec/year! Between the DSS and *Gaia*, we therefore expect that the position of the star has changed by about 0.5 arcmin. Let's see if this is the case! \n",
+    "\n",
+    "To start, we will query the *Gaia* catalog to retrieve data for this star (skip the cell below if you do not have an internet connection - we have provided the table locally as well). We use a large search radius so many sources will be returned"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Skip this cell if you are not connected to the internet\n",
+    "gaia_tbl = Gaia.query_object(SkyCoord.from_name('HD 219829'), \n",
+    "                             radius=1*u.arcmin)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the .read() below produces some warnings that we can safely ignore\n",
+    "with warnings.catch_warnings(): \n",
+    "    warnings.simplefilter('ignore', UserWarning)\n",
+    "    \n",
+    "    gaia_tbl = QTable.read('HD_219829_query_results.ecsv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We know that HD 219829 will be the brightest source in this small region, so we can extract the row with the smallest G-band magnitude. Let's check the proper motion values for this source to make sure that they are large:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hd219829_row = gaia_tbl[gaia_tbl['phot_g_mean_mag'].argmin()]\n",
+    "hd219829_row['source_id', 'pmra', 'pmdec']"
```

Comment:
This table looks weird on the rendered version:

![Screenshot_2020-11-22 Astronomical Coordinates 3 Working with Velocity Data in astropy coordinates — astropy-tutorials v3 0](https://user-images.githubusercontent.com/316517/99915087-c8157a00-2d01-11eb-9ac6-b10da7118d91.png)

Reply:
I think this is a bug with the current style - this should be resolved with the site design overhaul...